schema/swagger.yml

```diff
@@ -2588,191 +2599,3 @@ definitions:
-        maxLength: 64
-      broadhash:
-        type: string
-        example: 258974416d58533227c6a3da1b6333f0541b06c65b41e45cf31926847a3db1ea
-        format: hex
-      nonce:
-        type: string
-        example: sYHEDBKcScaAAAYg
-        minLength: 16
-        maxLength: 16
-
-  WSPeerUpdateRequest:
-    x-private: true
-    type: object
-    required:
-      - data
-      - socketId
-    properties:
-      data:
-        type: object
-        required:
-          - nonce
-        properties:
-          nethash:
-            type: string
-            maxLength: 64
-          broadhash:
-            type: string
-            format: hex
-          height:
-            type: integer
-            minimum: 1
-          nonce:
-            type: string
-            minLength: 16
-            maxLength: 16
-      socketId:
-        type: string
-      workerId:
-        type: number
-
-  WSSignaturesList:
-    x-private: true
-    type: object
-    required:
-      - nonce
-      - signatures
-    properties:
-      nonce:
-        type: string
-        example: sYHEDBKcScaAAAYg
-        minLength: 16
-        maxLength: 16
-      signatures:
-        type: array
-        items:
-          type: object
-        minItems: 1
-        maxItems: 25
-
-  WSBlocksList:
-    x-private: true
-    type: array
-    items:
-      type: object
-
-  WSBlocksCommonRequest:
-    x-private: true
-    type: object
-    required:
-      - ids
-    properties:
-      ids:
-        type: string
-        format: csv
-
-  WSTransactionsRequest:
-    x-private: true
-    type: object
-    required:
-      - nonce
-      - transactions
-    properties:
-      nonce:
-        type: string
-        example: sYHEDBKcScaAAAYg
-        minLength: 16
-        maxLength: 16
-      transactions:
-        type: array
-        items:
-          type: object
-        minItems: 1
-        maxItems: 25
-
-  WSAccessObject:
-    x-private: true
-    type: object
-    required:
-      - peer
-      - authKey
-      - updateType
-    properties:
-      peer:
-        $ref: '#/definitions/Peer'
-      authKey:
-        type: string
-      updateType:
-        type: integer
-        minimum: 0
-        maximum: 1
-
-  WSTransactionsResponse:
-    x-private: true
-    type: object
-    required:
-      - transactions
-    properties:
-      transactions:
-        type: array
-        uniqueItems: true
-        maxItems: 100
-        items:
-          type: object
-
-  WSSignaturesResponse:
-    x-private: true
-    required:
-      - signatures
-    properties:
-      signatures:
-        type: array
-        uniqueItems: true
-        maxItems: 100
-        items:
-          type: object
-
-  WSBlocksBroadcast:
-    x-private: true
-    type: object
-    required:
-      - block
-      - nonce
-    properties:
-      nonce:
-        type: string
-        example: sYHEDBKcScaAAAYg
-        minLength: 16
-        maxLength: 16
-      block:
-        $ref: '#/definitions/ReducedBlock'
```

Comment:
@Tschakki Why you removed this from specification? These schema's are necessary for the application.